src/main/java/com/internet/shop/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.internet.shop.service.impl;
+
+import com.internet.shop.dao.UserDao;
+import com.internet.shop.lib.Inject;
+import com.internet.shop.lib.Service;
+import com.internet.shop.model.User;
+import com.internet.shop.service.UserService;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Inject
+    private UserDao userDao;
+
+    @Override
+    public User create(User user) {
+        return userDao.create(user);
+    }
+
+    @Override
+    public User get(Long id) {
+        return userDao.get(id).orElseThrow(() ->
+                new IllegalArgumentException(" This " + id + " doesn't exist!"));
+    }
+
+    @Override
+    public List<User> getAll() {
+        return userDao.getAll();
+    }
+
+    @Override
+    public User update(User user) {
+        return userDao.update(user);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        try {
+            return userDao.delete(id);
+        } catch (IllegalArgumentException e) {
+            throw new RuntimeException(
+                    "Couldn't delete user with id " + id);
+        }
```

Comment:
This makes no sense.
We need to get all the users from DB, then filter them by required id, then delete if they exist, if not - return false.